### PR TITLE
Add Admission project

### PR DESCRIPTION
**Pull request by Abdo9826**
Add HTML file

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
```
Comment (Abdo9826) [inline index.html:80]: Please do not leave unused code as comments in your codebase, As comments make your code unclean

File: index.html [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+    <h1 id="title">Hello world, we are developers</h1>
+    <p id="description">For now, the test suite only works in Chrome!
+      Please read the README below in the JS Editor before beginning.
+      Feel free to delete this message once you have read it.
+      Good luck and Happy Coding!
+    </p>
+    <form id="survey-form">
+
+
```
Comment (Abdo9826) [inline index.html:20]: Please do not leave empty spaces between code lines

**Review by Abdo9826 — COMMENTED** [review]
requested changes on this pull request.

Hi <STUDENT_MENTION>,

Great work on making the changes 👏
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

**Review by Lucas-Masaba — APPROVED** [review]
# Hello @Abdo9826  👋 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

almost-there

## Highlights 🍾 
- Comments phrased politely 🏄 
- Explainations on why certain changes were required 👍 

## Optional suggestions 💭 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏 💻 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

[OPTIONAL]
- I suggest use of emojis in comments 👍 . An emoji is an equivalent to 20 words and they make the person receiving the review feel much better and at ease 🏄

**Pull request merged**